.github/workflows/run-tests.yml

```diff
@@ -10,6 +10,12 @@ jobs:
     name: Check and Test
     runs-on: ubuntu-20.04
 
+    services:
+      mongodb:
+        image: mongo:4.2.3
+        ports:
+          - 27018:27017
```

Comment:
don't we start mongo also with mongo runner?

Reply:
In the ci it was having trouble downloading it, so in the ci runs it doesn't use mongo runner, just the github image that exists. A bit hacky since it's testing two different environments but it's easy to configure so I think its ok?